src/components/Comment/Comment.tsx

```diff
@@ -529,20 +535,19 @@ export const Comment = ({
 								}}
 							/>
 							<div className={spaceBetween}>
-								<div className={shiftLeft}>
 									<Row>
 										{/* When commenting is closed, no reply link shows at all */}
 										{!isClosedForComments && (
 											<>
 												{/* If user is not logged in we link to the login page */}
 												{user ? (
-													<div className={svgOverrides} id="fgsghdiusfhfdsiu">
+													<div className={cx(svgOverrides, buttonOverrides)}>
 														<ButtonLink
 															pillar={pillar}
 															onClick={() => setCommentBeingRepliedTo(comment)}
 															icon={<SvgIndent />}
 															iconSide="left"
-															linkName="reply to comment"
+															linkName="reply-to-comment"
```

Comment:
This change seems unrelated. What's the reasoning for this?

Reply:
I think @liywjl preferred the linkName to be hyphenated over having spaces. I think it makes sense as that should keep it in line with other link names, but potentially it could be pulled out as a separate PR to keep the diff clean